content/tutorials/2.projects/use-directus-templates-in-clay-for-data-enrichment.md

```diff
@@ -0,0 +1,417 @@
+
+If you don't already have a Directus project, the easiest way to get started is with our [managed Directus Cloud service](https://directus.cloud).
+
+### Set Up Clay
+
+Make sure you have a Clay account and are familiar with:
+
+- Creating enrichment columns
+- Basic Clay workflow concepts
+- How to search for and use templates
+
+## How to Connect Directus and Clay
+
+There are two separate ways to connect Directus and Clay, each with different setup processes:
+
+**[Clay → Directus (HTTP API Templates)](#using-http-api-templates-clay-directus)**  
+Use Clay's pre-built HTTP API templates to pull data from Directus for enrichment or push enriched data back. Ideal for on-demand operations.
+
+**[Directus → Clay (Webhooks)](#using-webhooks-directus-clay)**  
+Use Directus Flows to automatically send data to Clay webhooks when events occur in your instance. Perfect for real-time data sync.
+
+![Directus Clay Integration Overview](/img/tutorials/directus_clay_integration.png)
+
+---
+
+## Using HTTP API Templates (Clay → Directus)
+
+Clay provides pre-built templates for common operations with Directus. These templates appear as enrichment column options and handle all the API configuration for you.
+
+Follow these steps to use the pre-built Directus templates in Clay.
+
+### Step 1: Set Up Authentication
+
+Before using any templates, configure your Directus authentication in Clay:
+
+1. Go to your Clay account settings
+2. Add a new HTTP API account
+3. Set up the authorization header:
+   - **Key:** `Authorization`
+   - **Value:** `Bearer YOUR_DIRECTUS_TOKEN_HERE`
+4. Name it "Directus" for easy reference
+
+Once created, you can select this account for all Directus API calls instead of manually entering headers each time.
+
+### Step 2: Add a Directus Template
+
+1. In your Clay table, create a new enrichment column
+2. Search for "directus" in the template search
+3. Select the template that matches your operation:
+   - **Create Item in Collection** - to add new records
+   - **Update Item in Collection** - to modify existing records
+   - **Get Item from Collection** - to search and retrieve records
+   - **Get Related Item Details** - to fetch relational data
+
+**Important:** The Directus templates use generic collection names (like "posts" or "users") as examples. You'll need to adapt these to match your specific Directus schema by replacing collection names, adjusting field names, and configuring filters based on your data structure.
+
+### Step 3: Configure the Template
+
+Each template requires configuration specific to your Directus setup:
+
+**Basic Configuration:**
+
+- **Directus URL:** Your instance URL (e.g., `https://your-project.directus.app`)
+- **Collection Name:** Replace "posts" with your actual collection name
+- **Account:** Select the "Directus" account you created in Step 1
+
+**For GET operations, add query parameters:**
+
+- **limit:** Maximum number of items to return (e.g., `10`)
```

Comment:
Should we use the fields prose component for these parameters?

https://ui.nuxt.com/docs/typography/field
https://ui.nuxt.com/docs/typography/field-group